shell/src/main/java/alluxio/shell/command/MountCommand.java

```diff
@@ -83,6 +86,21 @@ public Options getOptions() {
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
+    if (args.length == 0) {
+      Map<String, MountPointInfo> mountTable = mFileSystem.getMountTable();
+      String leftAlignFormat = " %-20s  %-3s %-60s %-8s %-15d %-15d %-5b %-5b   ";
+      for (Map.Entry<String, MountPointInfo> entry :
+              mountTable.entrySet()) {
+        String mMountPoint = entry.getKey();
+        MountPointInfo mountPointInfo = entry.getValue();
+        System.out.format(leftAlignFormat, mMountPoint, "on", mountPointInfo.getUfsUri(),
```

Comment:
I think it should be `mountPointInfo.getUfsUri(), "on", mMountPoint`,
for the following attributes, can you make the output look like:
"(local, capacity=xxx bytes, used bytes=xxxx bytes, not read-only, shared)"